pygfx/renderers/wgpu/_pipeline.py

```diff
@@ -261,7 +261,7 @@ def __init__(self, shader):
 
         # The info that the shader generates
         self.shader_hash = b""
-        self.resources = None
+        self.bindingss = None  # dict of dict of bindins, so multi-plural
```

Comment:
Typo

Reply:
Will also rename "bindings" to something more sensible.